package.json

```diff
@@ -109,15 +110,16 @@
         "terser-webpack-plugin": "^3.0.1",
         "webpack": "^4.39.3",
         "webpack-cli": "^3.3.5",
-        "webpack-merge": "^4.2.1"
+        "webpack-merge": "^4.2.1",
+        "worker-loader": "^3.0.5"
     },
     "browserslist": [
         "last 2 versions",
         "> 5%"
     ],
     "husky": {
         "hooks": {
-            "pre-commit": "yarn sanity-check && lint-staged"
```

Comment:
I am assuming you have removed that because (on some systems) this takes quite a while? :D I would still prefer to keep it as we did accidentally commit merge conflict artifacts in the past. @mal-tee already complained about the speed and is working on an implementation that can leverage ripgrep.

Reply:
Didn't work on Windows at all, didn't mean to commit this though